fuzz/fuzz_test.go

```diff
@@ -2,12 +2,17 @@ package pprof
 
 import (
 	"io/ioutil"
+	"runtime"
 	"testing"
 
 	"github.com/google/pprof/profile"
 )
 
 func TestParseData(t *testing.T) {
```

Comment:
Looks like you are reusing some old branch? Skipping TestHttpsInsecure on nacl is in master for a while - https://github.com/google/pprof/blob/9277d675724626f62433309e6dd3e8ec48a3e76e/internal/driver/fetch_test.go#L355